src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -1,19 +1,20 @@
 /** Tests to see if Find Nearest X button is appending correct text */
-
 import SimidMapCreative from './simid_map_creative.js';
 
 let testMap;
+let startData;
 
-beforeEach(() => {
-    testMap = new SimidMapCreative();
-    document.body.innerHTML = '<button id="findNearest"></button>';
-});
-
-test('testing button text updates from ad params', () => {
+/**
+ * Creates an object containing the test event data used in the SIMID protocol.
+ * @param {string=} adParameters Takes in the given string of AdParameters.
+ * If the field is left blank, passes in the default parameters.
+ * @return {!Object} an object containing the event data and corresponding AdParameters.
+ */
+function createEventData(adParameters = ""){
```

Comment:
this has type: "SIMID:Player:init"

Either change the name here to createInitData, or rework this function a bit so that it can be used for any event type

Reply:
changed